Formula/pdnsd.rb

```diff
@@ -24,6 +24,43 @@ def install
     system "make", "install"
   end
 
+  def caveats; <<-EOS.undent
+    Before starting "pdnsd" using "brew services" or manually, make sure that,
+    #{etc}/pdnsd.conf, #{var}/log/pdnsd.log and directory #{var}/cache/pdnsd
```

Comment:
> FWIW, I can still edit the configuration file with my account without the need for sudo, after which I have to chown

`#{etc}/pdnsd.conf` needs to be owned by `root` not `nobody`. But the above statement is still true.